### ISSUE TITLE
USWDS - Step Indicator Page: Not Completed Steps Have Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [uswds/uswds#4836]
## Description

On the Step Indicator page ( https://designsystem.digital.gov/components/step-indicator/ ), there is not enough color contrast for the uncompleted steps. Even in the "Usability Guidance section, it mentions, "Pending segments should be the least visually prominent of the segments, but should still maintain accessible contrast and not appear disabled." While the examples on the page have a color contrast ratio of 1.2:1 with a light gray bar ( #E6E6E6 ) on a white background, and the same applies to the borders around the step number for those items too.
